x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsIntegTests.java

```diff
@@ -153,6 +154,9 @@ public void testCreateAndRestoreSearchableSnapshot() throws Exception {
                 new ByteSizeValue(randomLongBetween(10, 100_000))
             );
         }
+        if (randomBoolean()) {
```

Comment:
Can we capture the number of replica that it randomly set and checks once the index is mounted that it corresponds?